radio/src/lua/api_general.cpp

```diff
@@ -1506,6 +1506,38 @@ static int luaResetGlobalTimer(lua_State * L)
   return 0;
 }
 
+/*luadoc
+@function multiBuffer(address[,value])
+
+This function reads/writes the Multi protocol buffer to interact with a protocol².
+
+@param address to read/write in the buffer
+@param (optional): value to write in the buffer
+
+@retval buffer value (number)
+
+@status current Introduced in 2.2.0
+*/
+#if defined(MULTIMODULE)
+extern uint8_t Multi_Buffer[];
+#define MULTI_BUFFER_SIZE 177
+
+static int luaMultiBuffer(lua_State * L)
+{
+  uint8_t address = luaL_checkunsigned(L, 1);
+  if(address>=MULTI_BUFFER_SIZE) {
+    lua_pushinteger(L, 0);
+    return 0;
+  }
+  uint16_t value = luaL_optunsigned(L, 2, 0x100);
+  if(value<0x100) {
+    Multi_Buffer[address]=value;
```

Comment:
here allocate the buffer if not already done

Reply:
Done